src/components/ebay-carousel/index.js

```diff
@@ -128,21 +151,26 @@ function onRender() {
         });
 
         if (config.nativeScrolling) {
-            const offset = getOffset(state);
+            const offset = hasOverride ? offsetOverride : getOffset(state);
+            // Listen for future scroll events and snap to the closest point.
+            this.cancelScrollHandler = onScrollEnd(listEl, handleScrollEnd.bind(this));
             // Animate to the new scrolling position and emit update events afterward.
             this.cancelScrollTransition = scrollTransition(listEl, offset, this.emitUpdate);
         }
 
+        if (autoplay && !paused) {
+            this.autoPlayTimeout = setTimeout(() => this.nextEl.click(), 4000);
```

Comment:
(minor) `autoplay` is used as 1 word elsewhere

Reply:
Will switch to all being `autoplay`. 👍